tensorflow/compiler/xla/service/allocation_tracker.h

```diff
@@ -63,10 +63,10 @@ class Allocation {
     CHECK_GE(ref_count_, 0);
     return ref_count_;
   }
-  void increment_ref_count() {
+  void increment_ref_count(int inc) {
     CHECK_GT(ref_count_, 0);
-    CHECK_LT(ref_count_, INT_MAX);
-    ++ref_count_;
+    CHECK_LT(ref_count_, INT_MAX-inc-1);
```

Comment:
I think this should be:

CHECK_LT(ref_count_, INT_MAX - inc + 1);

or perhaps more clearly:

CHECK_LE(ref_count_, INT_MAX - inc);

Agreed that this is largely of theoretical interest as reference counts should never get close to INT_MAX :-)